frontend/model/database.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
-import sbp from '~/shared/sbp.js'
 import localforage from 'localforage'
+import sbp from '~/shared/sbp.js'
```

Comment:
I've moved this import for two reasons:
- in principe, third-party module imports should come before our own imports
- when it's easy to do, lexical ordering is usually nicer than no ordering at all

Reply:
> in principe, third-party module imports should come before our own imports

Ah ok. Well, I've already made SBP into [a third-party module](https://www.npmjs.com/package/@sbp/sbp), it just hasn't been imported yet into this project. That's an open issue: #1153

So can we revert this change?

Reply:
Sure. This will still break lexical ordering though. Unless we insert a blank line after the `sbp` import:
```js
import sbp from '~/shared/sbp.js'

import localforage from 'localforage'

Reply:
This would even make it slightly more obvious that we're going to use SBP in this file.

Reply:
ok sure, it sounds like it's more important to you than it is to me, and it really isn't a big deal either way, so feel free to do it this way